cms/djangoapps/contentstore/rest_api/v0/tests/test_tabs.py

```diff
@@ -0,0 +1,216 @@
+""" Tests for the course tab API. """
+
+import json
+from urllib.parse import urlencode
+
+import ddt
+from django.urls import reverse
+from xmodule.modulestore.tests.factories import ItemFactory
+from xmodule.tabs import CourseTabList
+
+from cms.djangoapps.contentstore.tests.utils import CourseTestCase
+
+
+@ddt.ddt
+class TabsAPITests(CourseTestCase):
+    """Test cases for Tabs (a.k.a Pages) page"""
+
+    def setUp(self):
+        """Common setup for tests"""
+
+        # call super class to setup course, etc.
+        super().setUp()
+
+        # Set the URLs for tests
+        self.url = reverse(
+            "cms.djangoapps.contentstore:v0:course_tab_list",
+            kwargs={"course_id": self.course.id},
+        )
+        self.url_settings = reverse(
+            "cms.djangoapps.contentstore:v0:course_tab_settings",
+            kwargs={"course_id": self.course.id},
+        )
+        self.url_reorder = reverse(
+            "cms.djangoapps.contentstore:v0:course_tab_reorder",
+            kwargs={"course_id": self.course.id},
+        )
+
+        # add a static tab to the course, for code coverage
+        self.test_tab = ItemFactory.create(
+            parent_location=self.course.location,
+            category="static_tab",
+            display_name="Static_1",
+        )
+        self.reload_course()
+
+    def check_invalid_response(self, resp):
+        """Verify response is an error listing the invalid_tab_id"""
+
+        assert resp.status_code, 400
+        print("RESPONSE:", resp.content)
+        resp_content = json.loads(resp.content.decode("utf-8"))
```

Comment:
I assume `response.content` is a byte and you are converting it to a string, can you explain why? will `json.loads` fail on bytes?

Reply:
Nope, I just copied over this method from the existing tests and used it as is. Will remove.